index.js

```diff
@@ -182,24 +189,44 @@ Liftoff.prototype.launch = function (opts, fn) {
     return this.completions(completion);
   }
 
+  opts = mapValues(opts);
```

Comment:
What are we mapping for?

Reply:
I wanted to get a shallow-copy of `opts` because `opts.argv` is changed.
I'll modify this to use [shallow-clone](https://github.com/jonschlinkert/shallow-clone) or [extend-shallow](https://github.com/jonschlinkert/extend-shallow).